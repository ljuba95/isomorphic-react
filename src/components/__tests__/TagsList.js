import React from 'react'
import TagsList from '../TagsList'
import renderer from 'react-test-renderer'

describe('Tags List component', () => {
  it('renders as expected', () => {
    const tree = renderer
      .create(<TagsList tags={['js','haskell','go']}/>)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})