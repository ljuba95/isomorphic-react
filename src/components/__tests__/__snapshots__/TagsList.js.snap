// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags List component renders as expected 1`] = `
<div>
  <code>
    js 
  </code>
  <code>
    haskell 
  </code>
  <code>
    go 
  </code>
</div>
`;
